scripts/generate-entry-points.js

```diff
@@ -77,6 +53,31 @@ try {
         );
         await fs.writeFile(destinationFilePath, injectedHtml, 'utf-8');
     }
+    console.log(`Done!`);
 } catch (err) {
     console.error('Error generating entry points:', err);
 }
+
+/**
+ * Fetches the top 100 trade pairs by volume from the dYdX indexer API
+ * @returns {Promise<string[]>} Array of top trade pairs
+ * @throws {Error} If the API request fails
+ */
+async function fetchTopMarkets(url) {
+    const response = await fetch(url);
+    const data = await response.json();
+
+    // Convert markets object to array of entries and sort by volume
+    const sortedMarkets = Object.entries(data.markets)
+        .sort((a, b) => {
+            const volumeA = parseFloat(a[1].volume24H);
+            const volumeB = parseFloat(b[1].volume24H);
+            return volumeB - volumeA;
+        })
+        .slice(0, 100) // Take top 100
+        .map(([_, market]) => {
+            return market.ticker;
+        });
+
+    return sortedMarkets;
```

Comment:
small suggestion to potentially catch errors in response before we sort.

```suggestion
async function fetchTopMarkets(url) {
    const response = await fetch(url);
    if (!response.ok) {
        throw new Error(`HTTP error! status: ${response.status}`);
    }
    const data = await response.json();

    // Convert markets object to array of entries and sort by volume
    const sortedMarkets = Object.entries(data.markets)
        .sort((a, b) => {
            const volumeA = parseFloat(a[1].volume24H);
            const volumeB = parseFloat(b[1].volume24H);
            return volumeB - volumeA;
        })
        .slice(0, 100) // Take top 100
        .map(([_, market]) => {
            return market.ticker;
        });

    return sortedMarkets;
```

Reply:
If there is an error in fetching, an outer try-catch will catch it anyway... but sure, why not :)